src/spec/billing_spec.js

```diff
@@ -48,4 +47,98 @@ describe('Billing Page', () => {
     expect(plan).toBe('PLAN-60K');
     expect(billingPage.getPrice()).toBe('USD 31.80 x month');
   });
+
+  it('should show credit card icon when complete visa credit card number valid', () => {
+
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/settings/billing?plan=PLAN-60K');
+    browser.addMockModule('descartableModule2', () => angular
+      // This code will be executed in the browser context,
+      // so it cannot access variables from outside its scope
```

Comment:
I think that it is not necessary to copy this comment, only take it into account.